web3.md

```diff
@@ -1182,6 +1199,11 @@ Transaction Receipts
          <k> #halt ~> #catchHaltTx ACCT => #mkCodeDeposit ACCT ... </k>
       requires ACCT =/=K .Account
 
+    syntax KItem ::= "#setPC" Int
+ // -----------------------------
+    rule <k> #setPC PCOUNT => . ... </k>
+         <pc> _ => PCOUNT </pc>
+
```

Comment:
It feels hacky to set the program counter to get the returned opcode position. Are there any other ways we can do this? We could, for example, change `EVMC_REVERT` to store the program counter that the revert happens at (similarly for `EVMC_INVALID`).

Reply:
This is similar to the use of `#refund` to preserve the gas cell, so it seemed ok to me. I don't see the `EVMC_INVALID` example you're talking about, can you point it to me?